### ISSUE TITLE
Thank you for this project

### ISSUE THREAD

**Issue by apjyotirmay** [florisboard/florisboard#126]
Hi, I've always been on a lookout for an alternative to GBoard, something that's as neat looking as that and isn't a huge compromise. Saw this project on F-droid today, and it's really good!

Thank you for your time and effort and making it open source. You're amazing!

**Comment by mirsella** [comment]
while this issue is here, i'm saying thanks too.

 i wanted to use colemak-DHM (colemak variant), which is on 0 keyboard app. so i tried openboard but the codebase was hell to work with, just to add a new layout. discovered this project on r/android and it was straightforward and easy to add a [new layout](https://github.com/mirsella/florisboard)

also tell me if i open another issue :
on gboard for example when using the shift key to write a letter in CAPITAL letters, others keys change like on azerty the ' become a ? i think it would be nice to have a option for this, like in the layout.json file something like : 
`      { "code":   39, "label": "'", "\""`

**Comment by ainola** [comment]
Yeah, this is a seriously well-thought-out keyboard. I switched pretty much instantly when I saw it im Fdroid as well.

**Comment by patrickgold** [comment]
Thanks for your feedback, I really appreciate it! My main inspiration for FlorisBoard was/is to have a good alternative to proprietary keyboards such as Gboard in terms of functionality while fully respecting the users privacy like other open-source keyboards do. I am constantly trying to improve FlorisBoard (I am reworking the theme settings + engine currently), which will bring more stability and customization options to the design of FlorisBoard.

@mirsella I am somewhat aware of this issue (noticed it while implementing Colemak and Dvorak), but chose then to not support it for the time being. I am open though for adding a new field to support such actions. If possible, it would be best if you created a new issue for this, thank you!

**Comment by kj7rrv** [comment]
Same from me. I'm currently trying to degooglify my phone, and this is the first keyboard other than Gboard that I've really liked.

**Comment by xtyrrell** [comment]
I also want to make heard my appreciation for this project and all the work put in. Thank you @patrickgold, you are a legend and an excellent steward! And thanks to all the other contributors too!

**Comment by another-sapiens** [comment]
Just noticed this thread and wanted to join in. I've said this before and I'll say it again: This is the best keyboard app ever. Period. 

The key feature here is privacy, but now that it has suggestions and the new swipe typing, I really don't miss anything from the propietary ones. 
At first I was kinda concerned it wouldn't be as refined as Gboard, for example, and that the transition would be hard, but dang I was wrong. Specially with the gliding, thought it would be impossible, but oh boy is it working nicely. 

Also, I love how our suggestions really are taken into account around here. Really makes it possible to compete against the big boys in the app playground

@patrickgold thank you for all your effort in the name of open source software, and in the name of privacy. 
And thank you, contributors [looking at you @X-yl ;)], or anyone that helps making FOSS a viable alternative for everyday use.

I wanted to help translate it to spanish, but it seems like it's already done. 
Anyways, I'd better stop writing, I don't wanna waste anyone's time.
Dev, you rock!

**Comment by opusforlife2** [comment]
Adding my thanks as well!

Like everyone else, I've been looking for an open source replacement for Google Keyboard for years (yes, that's right, I downgraded from GBoard pretty soon after I got updated to that shitty version) but no app has been able to make the cut. Specifically, no app had good enough swipe/glide typing, which was a deal breaker. I've had to keep the proprietary Google Swipe libs around, which is a roadblock in the goal to going fully open source.

But Floris's swipe typing is really good! It still takes _slightly_ more concentration than Google Keyboard as of version 0.3.12, which I think is pretty great for an app that hasn't even reached version 1.0 yet.

The point is, this app is showing enough potential that I'm thinking of recommending it to regular people (not interested in FOSS, that is), which is a great compliment for any FOSS app.

Looking forward to seeing this app grow. <3
